packages/dm-core-plugins/src/index.tsx

```diff
@@ -1,71 +1,108 @@
 import { TUiPluginMap } from '@development-framework/dm-core'
 
-import { YamlPlugin } from './yaml/YamlPlugin'
-import { BlueprintHierarchyPlugin } from './blueprint-hierarchy/BlueprintHierarchyPlugin'
-import { JobPlugin } from './job/JobPlugin'
-import { FormPlugin } from './form/FormPlugin'
-import HeaderPlugin from './header/HeaderPlugin'
-import JsonPlugin from './json/JsonPlugin'
-import { PdfPlugin } from './pdf/PdfPlugin'
-import { BlueprintPlugin } from './blueprint/BlueprintPlugin'
-import ExplorerPlugin from './explorer/ExplorerPlugin'
-import { GridPlugin } from './grid/GridPlugin'
-import { ListPlugin } from './list/ListPlugin'
-import { TablePlugin } from './table/TablePlugin'
-import { FilePlugin } from './file/FilePlugin'
-import { RoleFilterPlugin } from './role_filter/RoleFilterPlugin'
-import { SidebarPlugin } from './view_selector/SidebarPlugin'
-import { TabsPlugin } from './view_selector/TabsPlugin'
+import { lazy } from 'react'
 
 export { WidgetProvider } from './form/context/WidgetContext'
 
 export default {
   '@development-framework/dm-core-plugins/explorer': {
-    component: ExplorerPlugin,
+    component: lazy(() => import('./explorer/ExplorerPlugin')),
   },
   '@development-framework/dm-core-plugins/list': {
-    component: ListPlugin,
+    component: lazy(() =>
+      import('./list/ListPlugin').then((module) => ({
+        default: module.ListPlugin,
+      }))
```

Comment:
Very cool :D 
Why  is the syntax different in these plugins compared to "explorer"?
Looks different from the react docs as well